### PR TITLE
56629 migrate ombinfo component

### PR DESCRIPTION
**Pull request by evansmith**
## Summary

- Removed the OMBInfo Component and replaced it with the va-omb-info web component. Updated code per the linter suggestions.
- No Bugs related to this issue.
- This is the updated way to show web components per this documentation: https://design.va.gov/about/developers/using-web-components#how-to-migrate-to-web-components
- VA Benefits and Claims
- No Flipper used.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
[department-of-veterans-affairs/va.gov-team#0000](https://github.com/department-of-veterans-affairs/va.gov-team/issues/56629)
- _Link to previous change of the code/bug (if applicable)_
N/A
- _Link to epic if not included in ticket_
N/A

## Testing done

- The old version had a link while the new version has a button. Confirmed the va-button opened the same modal as the previous link. Added an E2E test to confirm that the va-button and va-omb-info components were present.
- Go to /view-change-dependents/add-remove-form-21-686c/introduction and scroll to where the text includes: Respondent Burden. Confirm clicking View Privacy Act Statement opens a modal.
- Tested locally on Chrome, Safari, and Edge. Created a new E2E test to verify the component's existence.
- Used Chrome Axe plugin for Axe tests.
- Used Apple Screen Reader to confirm accessibility.
- Used Colour Contrast Analyser to confirm contrast tests.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  | Before Mobile OMB Component |  After Mobile OMB Component|
| Desktop |  Before OMB Component | After OMB Component|

## What areas of the site does it impact?

No areas are impacted outside of the introduction page of 686c.

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/disability-benefits/686c-674/containers/IntroductionPage.jsx [hunk 1]
```diff
@@ -128,10 +127,10 @@ class IntroductionPage extends React.Component {
...
           />
           <div className="omb-info--container vads-u-padding-left--0">
-            <OMBInfo
-              resBurden={30}
-              ombNumber="2900-0043"
-              expDate="09/30/2021"
+            <va-omb-info
+              res-burden={30}
+              omb-number="2900-0043"
+              exp-date="09/30/2021"
```
Comment (tblackwe) [inline src/applications/disability-benefits/686c-674/containers/IntroductionPage.jsx:133]: What is this date? it reads like a past expiration

Comment (evansmith) [reply src/applications/disability-benefits/686c-674/containers/IntroductionPage.jsx:133]: @tblackwe I'm not sure, a look through the code shows a few that would be past expiration and a few that haven't expired yet.

Comment (tblackwe) [reply src/applications/disability-benefits/686c-674/containers/IntroductionPage.jsx:133]: definitely an expiration date. https://design.va.gov/storybook/?path=/docs/components-va-omb-info--children

suggest checking with design/other engineers to determine if this should be updated.

Comment (tblackwe) [reply src/applications/disability-benefits/686c-674/containers/IntroductionPage.jsx:133]: orrrr.... We could just pull it off the 674 form.

> OMB Approved No. 2900-0049
> Respondent Burden: 15 minutes
> Expiration Date: 10/31/2024

https://www.vba.va.gov/pubs/forms/vba-21-674-are.pdf

**Review by tblackwe — APPROVED** [review]

File: src/applications/disability-benefits/686c-674/containers/IntroductionPage.jsx (hunk 1 above)
Comment (evansmith) [reply src/applications/disability-benefits/686c-674/containers/IntroductionPage.jsx:133]: @tblackwe I updated the control number and it's ready for another review

Comment (evansmith) [reply src/applications/disability-benefits/686c-674/containers/IntroductionPage.jsx:133]: @tblackwe disregard that, it should stay at 0043, i was looking at the 674 form. pushed it back.

**Review by navaeddie — APPROVED** [review]

**Pull request merged**